skype/skype-ps/skype/Set-CsTeamsMeetingPolicy.md

```diff
@@ -1072,6 +1083,20 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowNetworkConfigurationSettingsLookup
+Determines whether network configuration setting lookup can be made for users who are not enterprise voice enabled, it will be used for enabling network roaming policy
```

Comment:
```suggestion
Determines whether network configuration setting lookup can be made for users who are not Enterprise Voice enabled; it will be used for enabling network roaming policy.
```